WebAPI/CurrencyConverter/Data/ExchangeRateRepository.cs

```diff
@@ -38,9 +38,19 @@ public async Task<ExchangeRate> GetAsync(int id)
         {
             return await _context.Set<ExchangeRate>().FindAsync(id);
         }
-        public async Task<ExchangeRate> GetByNameAsync(string currencyName)
+        public async Task<ExchangeRate> GetByNameAsync(string baseCurrencyName, string currencyName)
```

Comment:
Name of method doesn't reflect it's signature - GetByNAME but we use two parameters.